src/Infrastructure.Dapper/Repositories/UserRepository.cs

```diff
@@ -428,6 +428,55 @@ public UpdateUserData SetKeyConnectorUserKey(Guid userId, string keyConnectorWra
         };
     }
 
+    public UpdateUserData SetMasterPassword(Guid userId, MasterPasswordUnlockData masterPasswordUnlockData,
```

Comment:
Same comment about naming applies here, it is not clear by the naming which master password hash is required here, which may lead to confusion / bugs upon revisit.